### PR TITLE
refactor: notVotedNeurons -> votableNeurons

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Adjustment after refactoring in nns-js

# Changes

- renaming: notVotedNeurons -> votableNeurons

# Tests

no related tests

### CODE REVIEW

File: frontend/svelte/src/lib/components/proposal-detail/VotingCard/VotingCard.svelte [hunk 1]
```diff
@@ -16,7 +16,7 @@
   export let proposalInfo: ProposalInfo;
 
   const votableNeurons = () =>
-    getNotVotedNeurons({
+    getVotableNeurons({
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/components/proposal-detail/VotingCard/VotingCard.svelte:19]: you did not want to remove also the `.filter(({ votingPower }) => votingPower > 0);` in this component?

Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/components/proposal-detail/VotingCard/VotingCard.svelte:19]: yep, it should be part of the `getVotableNeurons`, right?

**Review by lmuntanerDfinity — COMMENTED** [review]
One question

File: frontend/svelte/src/lib/components/proposal-detail/VotingCard/VotingCard.svelte (hunk 1 above)
Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/proposal-detail/VotingCard/VotingCard.svelte:19]: Done 🤦‍♂️

Comment (mstrasinskis) [reply frontend/svelte/src/lib/components/proposal-detail/VotingCard/VotingCard.svelte:19]: Thanks for noticing

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks!

**Pull request merged**